### ISSUE TITLE
Implement GUI for use cases in the power management context area 

### ISSUE THREAD

**Issue by stefanieschreiner** [i315315/amos2022ws02-automotive-test-app#26]
## User story
1. As a User
2. I want to see a list of possible use cases in the context of power management 
3. So that it is possible to test power management use cases

## Acceptance criteria
 * In the power management context there is a list of all available use cases (currently Display off, Standby Mode) 
 * The list should be easily extendable.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
